rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/LRARecord.java

```diff
@@ -424,7 +424,9 @@ private boolean afterLRARequest(URI target, String payload) {
         Client client = null;
 
         try {
-            client = ClientBuilder.newClient();
+            client = ClientBuilder.newBuilder()
+                    .readTimeout(PARTICIPANT_TIMEOUT, TimeUnit.SECONDS)
```

Comment:
This method was added in a later revision of the JAX-RS spec (and was the reason I chose async). Will you research whether or not we need to support any of the earlier versions?

Reply:
Hm... I thought that we are fine to use the JAX-RS 2.1 when we defined it in dependencies.
Regarding the need of  support of the earlier versions. I can't see any need for the support or earlier versions. The Quarkus uses the 2.1 from start, for WildFly there will be for sure no support with older versions (at least for the reason that for integration there will be created an extension and WildFly supports extensions only in new versions).

Reply:
Why would we want to restrict our user base without any significant discernible benefit?

Reply:
So if you can indicate why the benefit of making this change outweighs the drawback of restricting our user base then I'll mark it as approved.

Reply:
Also bear in mind that I want to move to a reactive solution in a later revision so your changes would ultimately be rolled back.

Reply:
Also since the config property and the implementation technique of using async calls are independent of each I would recommend that you split up the issue to avoid one blocking the other.